agents/agent.json

```diff
@@ -24,7 +24,7 @@
   },
   "defaultTimezone": "Australia/Sydney",
   "webhook": {
-    "url": "https://\u003cyour-project-id\u003e.appspot.com/webhook/",
+    "url": "https://steelconnect-199911.appspot.com/webhook/",
```

Comment:
I think we don't want to change this in the template. Maybe just revert this file.